tests/json_schema_test_suite_test.rs

```diff
@@ -1,44 +1,120 @@
+                .tests
+                .into_iter()
+                .enumerate()
+                .map(move |(test_idx, test)| {
+                    let name = format!("{name}_{schema_idx}_{test_idx}");
+                    let schema = schema_test.schema.clone();
+                    let ignore = ignore_regex.is_match(&name);
+                    Trial::test(name, move || {
+                        translation_typecheck_test(schema, test.data, test.valid);
+                        Ok(())
+                    })
+                    .with_ignored_flag(ignore)
+                })
+        })
+        .collect()
+}
+
+const SKIP_TESTS: &[&str] = &[
+    "optional\\/format.*",
     "definitions_0_1.*", // contains an external reference (remote URI)
     "id_.*",
-    "maxItems_1_.*", // schemars doesn't accept floats as the value of `maxItems`
-    "maxLength_1_.*", // schemars doesn't accept floats as the value of `maxLength`
+    "maxItems_1_.*",      // schemars doesn't accept floats as the value of `maxItems`
+    "maxLength_1_.*",     // schemars doesn't accept floats as the value of `maxLength`
     "maxProperties_1_.*", // schemars doesn't accept floats as the value of `maxProperties`
-    "minItems_1_.*", // schemars doesn't accept floats as the value of `minItems`
-    "minLength_1_.*", // schemars doesn't accept floats as the value of `minLength`
+    "minItems_1_.*",      // schemars doesn't accept floats as the value of `minItems`
+    "minLength_1_.*",     // schemars doesn't accept floats as the value of `minLength`
     "minProperties_1_.*", // schemars doesn't accept floats as the value of `minProperties`
-    "optional_bignum_.*",
-    "optional_content_.*",
-    "optional_cross_draft_.*",
-    "optional_ecmascript_regex_.*",
+    "optional/bignum.*",
+    "optional/content.*",
+    "optional/cross-draft.*",
+    "optional/ecmascript-regex.*",
     "refRemote_.*", // no.
     // TODO: make reference handling robust
     // The following are references that aren't yet handled by js2n (remote URIs, local files
     // and non-top level definitions)
-    "ref_0_3.*", // reference to the whole schema `#` not yet supported
+    "ref_0_3.*",  // reference to the whole schema `#` not yet supported
     "ref_12_1.*", // reference to bare URI `node` (no fragment, no leading slash). Should fail
-                  // because invalid, but js2n replace that by a `Dyn` contract
+    // because invalid, but js2n replace that by a `Dyn` contract
     "ref_14_1.*", // reference to an anchor "#foo" not yet supported
     "ref_15_1.*", // anchor + remote URI
     "ref_16_1.*", // external reference (remote URI)
     "ref_18_0.*", // reference to a local definition ("#/definitions/inner" but the definition
-                  // isn't at the top-level? I'm not even sure how it should be handled in all
-                  // generality) + absolute local URI
+    // isn't at the top-level? I'm not even sure how it should be handled in all
```

Comment:
Nitpick: the new align is a bit strange. I think we should either put all the comments on their own line, or keep the old alignment. But starting inline/indented and then jumping to the next line unindented is harder to read, IMHO

Reply:
Fair enough. This was a macro but isn't anymore, so I guess rustfmt has opinions about that. I put them on their own lines.